src/odemis/util/img.py

```diff
@@ -777,3 +777,18 @@ def _getBoundingBox(content):
         md_pos[1] - half_shape_wc[1],
     )
     return rect
+
+
+def getOutliers(data, outliers=0):
```

Comment:
Let's move it next to `findOptimalRange()`, as it's just a wrapper to that function.